src/ConformalTracking.cc

```diff
@@ -1149,8 +1152,13 @@ void ConformalTracking::buildNewTracks(UniqueKDTracks& conformalTracks, SharedKD
               break;
 
             // Otherwise replace the existing track
+            // Assign kalman fit direction based on the parameters
+            bool kalmanFitBackward = parameters._kalmanFitBackward;  // true if backward is set
+            conformalTrack->setKalmanFitDirection(
```

Comment:
this should be `bestTrack->setKalmanFitDirection,` because `conformalTrack` is overwritten in line 1159.

Reply:
done